test/require-test.js

```diff
@@ -0,0 +1,47 @@
+var path = require('path');
+var temp = require('temp');
+var mocha = require('../');
+var fs = require('fs');
+var assert = require('assert');
+
+var tempFile, filePrefix = './';
+
+function removeFile(file) {
+  if (file && fs.existsSync(file)) { 
```

Comment:
Look at the `fs.existsSync` docs and you'll see why this is wrong. It might cause race issues. Better to just unlink in an unhandled try/catch.